src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -399,7 +398,8 @@ async def _consume_request_iterator(self,
                                         request_iterator: RequestIterableType
                                        ) -> None:
         try:
-            if inspect.isasyncgen(request_iterator):
+            if inspect.isasyncgen(request_iterator) or hasattr(
+                    request_iterator, '__aiter__'):
```

Comment:
Good catch.

Reply:
As you said in another way before, CPython sometimes sxxxx.